### PR TITLE
[Ready] High alarm ERT suits

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
#8256

:cl: Coolgat3
add: Added a better armor for Amber and Red alert ERT
tweak: Made Amber alert ERT spawn Red alert equipped ERT
imageadd: Added sprites for Amber/Red ERT armors and made muzzled, digitigrade leg, and lazy naga versions of those.
:cl:/

[why]: Alright so:
Adding the ERT is a cool thing, literally why not, I don't think there's any problem with that. They look cool, High alert ERTs should be armored more than the low alert ones, no?

Why would Amber spawn Red alert ERTs? Well, see. Amber alert is stupid, it is supposed to be something like code red, appearently? Or is it more towards blue? Literally what is amber alert? I feel like that alert is something of the likes of "We will all die!" While red is "We are all dying!" This is why I think Amber alert ERT should spawn Red alert ERT, We rarely have code red anymore because nobody is bothered to swipe the things, they rather just go amber and leave it be, therefore we never really see red alert ERT at all. I think making Amber alert ERT also a high alert response team is a good solution, besides, ERT is admin spawn so there's not really much talk about "oh yeah this is op" because an admin can choose whether to give that ERT those 2 pulse carbines and slightly better armor, right..?

So that'd be pretty much it, except the fact that making naga and taur sprites is literally the most damn terrible thing I've been put up against. I made the naga sprites only have the suit from waist up, so let's just all assume that ERT nagas have spaceproof tails until I get the vein to actually do it and manage to make it not look like utter garbage, because I am TERRIBLE at spriting characters and already spent like 5 hours on just doing the muzzles for the helmets, I shit you not.
Or if someone wants to do those version themself and are skilled in character spriting then hell, go ahead.

EDIT: Forgot to add sprite pictures.
https://i.imgur.com/DobkSqp.png This is the digitigrade and muzzled version.

The other sprites are basically what TG ert is. The naga sprites are just those with no legs so I don't think I really need to post them.

### CODE REVIEW

**Comment by Coolgat3** [comment]
Thank fuck now someone please legit review it and merge it when possible or else i'm gonna kms because of the fact that it'll sit here for a month (y)

**Comment by Trilbyspaceclone** [comment]
****DO NOT FULLY MERG****
Test Merg When Ready

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -182,7 +183,7 @@ Contains:
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert
 	allowed = list(/obj/item/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/tank/internals)
 	armor = list("melee" = 65, "bullet" = 50, "laser" = 50, "energy" = 50, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 80)
-	slowdown = 0
+	slowdown = 0 //Huh
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: un needed comment

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 2]
```diff
@@ -229,6 +230,75 @@ Contains:
...
+
+/obj/item/clothing/suit/space/hardsuit/ert/alert
+	name = "emergency response team suit"
+	desc = "Red alert command suit for the ERT. This one is more armored than its standard version."
+	icon_state = "ert_command-alert"
+	item_state = "ert_command-alert"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/alert
+	allowed = list(/obj/item/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/tank/internals)
+	armor = list("melee" = 70, "bullet" = 55, "laser" = 50, "energy" = 50, "bomb" = 65, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	slowdown = 0 //Huh
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: bad copy pasta

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 3]
```diff
@@ -229,6 +230,75 @@ Contains:
...
+	icon_state = "hardsuit0-ert_security-alert"
+	item_state = "hardsuit0-ert_security-alert"
+	item_color = "ert_security-alert"
+
+/obj/item/clothing/suit/space/hardsuit/ert/alert/sec
+	desc = "Red alert security suit for the ERT. This one is more armored than its standard version."
+	icon_state = "ert_security-alert"
+	item_state = "ert_security-alert"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/alert/sec
+	tauric = TRUE		//Citadel Add for tauric hardsuits
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: Uneeded, it's in the parent suit

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 4]
```diff
@@ -229,6 +230,75 @@ Contains:
...
+	icon_state = "hardsuit0-ert_engineer-alert"
+	item_state = "hardsuit0-ert_engineer-alert"
+	item_color = "ert_engineer-alert"
+
+/obj/item/clothing/suit/space/hardsuit/ert/alert/engi
+	desc = "Red alert engineer suit for the ERT. This one is more armored than its standard version."
+	icon_state = "ert_engineer-alert"
+	item_state = "ert_engineer-alert"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/alert/engi
+	tauric = TRUE		//Citadel Add for tauric hardsuits
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: ditto

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 5]
```diff
@@ -229,6 +230,75 @@ Contains:
...
+	item_state = "hardsuit0-ert_medical-alert"
+	item_color = "ert_medical-alert"
+
+/obj/item/clothing/suit/space/hardsuit/ert/alert/med
+	desc = "Red alert medical suit for the ERT. This one is more armored than its standard version."
+	icon_state = "ert_medical-alert"
+	item_state = "ert_medical-alert"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/alert/med
+	species_exception = list(/datum/species/angel)
+	tauric = TRUE		//Citadel Add for tauric hardsuits
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: ditto

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 6]
```diff
@@ -229,6 +230,75 @@ Contains:
 	species_exception = list(/datum/species/angel)
 	tauric = TRUE		//Citadel Add for tauric hardsuits
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/miscellaneous.dm]: should just clean up my laziness while you're at it. :P

**Review by Poojawa — CHANGES_REQUESTED** [review]

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 6 above)
Comment (Trilbyspaceclone) [reply code/modules/clothing/spacesuits/miscellaneous.dm]: <3

**Review by silicons — APPROVED** [review]

**Pull request merged**